global.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "projects": [ "src", "test" ],
+  "sdk": {
+    "version": "1.0.0-preview2-003121"
```

Comment:
I do see 1.0.0-rc1-final for `Microsoft.AspNet.WebUtilities` but that's the latest currently available.